lib/DDG/Meta/Helper.pm

```diff
@@ -64,6 +65,17 @@ Use booleans true and false to set options.
 
     $stash->add_symbol('&true', sub { 1 });
     $stash->add_symbol('&false', sub { 0 });
+
+=keyword rand_int
+
+generates random integer between 0 and supplied max integer (max defaults to 1).
+
+=cut
+
+	$stash->add_symbol('&rand_int', sub { 	
+		my $max = shift || 1;
```

Comment:
@nilnilnil @cngarrison I think `1` is a pretty sensible default - I've worked in a few languages where the general default for floating-point bounds is `0` and `1`, so it shouldn't cause *too* much confusion.